_includes/head.html

```diff
@@ -14,7 +14,23 @@
     <link rel="mask-icon" sizes="any" href="/images/opensourcedesign-logo.svg" color="#000">
 
     <link href="/css/main.css" rel="stylesheet">
-
+    
+    <!-- Facebook metadata -->
+    <meta property="og:url" content="https://opensourcedesign.net">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Open Source Design">
+    <meta property="og:image" content="https://avatars3.githubusercontent.com/u/4183553">
+    <meta property="og:description" content="Open Source Design1">
+    <meta property="og:site_name" content="Open Source Design">
+    <meta property="og:locale" content="en_US">
+    
+    <!-- Twitter metadata -->
+    <meta name="twitter:card" content="Open Source Design!">
+    <meta name="twitter:url" content="https://opensourcedesign.net">
+    <meta name="twitter:title" content="Open Source Design">
+    <meta name="twitter:description" content="Open Source Design!">
+    <meta name="twitter:image" content="https://avatars3.githubusercontent.com/u/4183553">
```

Comment:
Twitter doesn't have "twitter:image" or "twitter:title", etc.

Reply:
Huh, that's odd.  I read it in a guide but I will remove it.

Reply:
Alright, I'll give it a read once I'm off school internet.